src/index.ts

```diff
@@ -282,6 +282,7 @@ const getInputFilesFromReadme = (readMePath: string): asyncIt.AsyncIterableEx<Sp
 
     yield* openApiMd
       .getInputFiles(m.markDown)
+      .map(f => f.replace('$(this-folder)', '.'))
```

Comment:
Does it search from current folder by default without explicitly write it?

Reply:
Yes. It searches from the readme file parent directory.  So the path here is correct.